src/content/en/fundamentals/performance/rail.md

```diff
@@ -339,8 +338,8 @@ The following audits are especially relevant:
 [SW]: /web/tools/lighthouse/audits/registered-service-worker
 [3G]: /web/tools/lighthouse/audits/fast-3g
 [FMP]: /web/tools/lighthouse/audits/first-meaningful-paint
-[FI]: /web/tools/lighthouse/audits/first-interactive
-[CI]: /web/tools/lighthouse/audits/consistently-interactive
+[FCI]: /web/tools/lighthouse/audits/first-interactive
+[TTI]: /web/tools/lighthouse/audits/consistently-interactive
```

Comment:
there's already a TTI definition defined elsewhere and it points to a different URL, probably have to go with a different name here.

Reply:
Good catch. Next commit will include the change for this.